app/assets/json/translations/locale-en.json

```diff
@@ -763,6 +771,7 @@
                 "visit_a_housing_counselor": "Visit a local housing counselor for help with your application.",
                 "visit_homeownership_sf": "Visit HomeownershipSF"
             },
+            "at_total_income": "at <strong>{{income}}</strong>/{{per}}",
```

Comment:
same here, let's ditch the "/" while we're at it and have the per be "per month" or "per year" then we can reuse the "per_month" and "per_year" keys we already have too.